dolphin/api/validation/parameter_types.py

```diff
@@ -172,3 +172,22 @@ def valid_char(char):
     'minimum': 0, 'maximum': 65535
 }
 
+boolean = {
+    'type': ['string'],
+    'enum': [True, 'True', 'TRUE', 'true', False, 'False', 'FALSE', 'false'],
```

Comment:
Do we really need to support many combinations, lets think

Reply:
I think yes, I check cinder, it used the same way.